othello/apps/games/tasks.py

```diff
@@ -13,46 +14,47 @@
 from ...moderator import INITIAL_BOARD
 from ...moderator.moderator import InvalidMoveError, Moderator
 from ...moderator.runners import PlayerRunner, ServerError, UserError, YourselfRunner
-from ..games.models import Game, Player, Submission
+from ..games.models import Game, Player, Submission, GameLog, GameError, Move
 
 logger = logging.getLogger("othello")
 task_logger = get_task_logger(__name__)
 
 
-def send_through_game_channel(game, event_type, object_id):
+def send_through_game_channel(game: Game, event_type: str, object_id: int) -> int:
     task_logger.debug(f"sending {event_type}")
     async_to_sync(get_channel_layer().group_send)(
         game.channels_group_name, {"type": event_type, "object_id": object_id}
     )
 
 
-def check_heartbeat(game):
+def check_heartbeat(game: Game) -> bool:
     if game.is_tournament:
         return True
     game.refresh_from_db()
     return (timezone.now() - game.last_heartbeat).seconds < settings.CLIENT_HEARTBEAT_INTERVAL * 2
 
 
-def delete_game(game):
+def delete_game(game: Game) -> None:
     if not game.is_tournament:
         game.delete()
 
 
 @shared_task
-def run_game(game_id):
+def run_game(game_id: int) -> Any[str, None]:
     try:
-        game = Game.objects.get(id=game_id)
-        yourself = Submission.objects.get(user__username="Yourself")
+        game: Game = Game.objects.get(id=game_id)
+        yourself: Submission = Submission.objects.get(user__username="Yourself")
     except Game.DoesNotExist:
         logger.error(f"Trying to play nonexistent game ({game_id}")
         return "game not found"
 
-    mod, time_limit = Moderator(), game.time_limit
-    game_over = False
-    file_deleted = None
+    mod: Moderator = Moderator()
+    time_limit: int = game.time_limit
+    game_over: bool = False
+    file_deleted: GameError = None
 
     try:
-        black_runner = (
+        black_runner: Any[YourselfRunner, PlayerRunner] = (
```

Comment:
NO NO NO NO NO. You want `Union`. https://docs.python.org/3/library/typing.html#typing.Any

Besides, you don't need to annotate this..